content/02.main-text.md

```diff
@@ -152,6 +152,7 @@ Pandoc supports conversion between additional formats — such as LaTeX, AsciiDo
 [Journal Article Tag Suite](https://jats.nlm.nih.gov/) (JATS) is a standard XML format for scholarly articles that is used by publishers, archives, and text miners [@url:http://www.niso.org/standards/z39-96-2015/; @doi:10.6087/kcse.2014.1.99; @doi:10.1080/00987913.2012.10765464].
 Pandoc's JATS support provides an avenue to integrate Manubot with the larger JATS ecosystem.
 For now, the primary Manubot output is HTML intended to be viewed in a web browser.
+Manuscripts are typically submitted to journals using the DOCX format.
```

Comment:
Should we note that PDF is generally the submission format until a source submission is required at which point DOCX is used. Something like

> Authors generally use the PDF output for preprint or initial journal submissions. For journal submissions that do require an editable source document, Manubot's DOCX output is usually sufficient.